config/generic/make_defs.mk

```diff
@@ -70,8 +70,11 @@ CKVECFLAGS     :=
 else
 ifeq ($(CC_VENDOR),clang)
 CKVECFLAGS     :=
+ifeq ($(CC_VENDOR),nvc)
```

Comment:
There is a logic bug somewhere here which is breaking builds with `clang`.

Reply:
Sorry for the delay, it's fixed now.